pkg/controller/tektoninstallation/tekton.go

```diff
@@ -38,7 +40,7 @@ func NewSubscription(ns string) *olmv1alpha1.Subscription {
 		Spec: &olmv1alpha1.SubscriptionSpec{
 			Channel:                "dev-preview",
 			Package:                "openshift-pipelines-operator",
-			StartingCSV:            "openshift-pipelines-operator.v0.5.2",
+			StartingCSV:            StartingCSV,
```

Comment:
Value for StartingCSV is `eclipse-che.v7.2.0`, not sure why both Che and Tekton set with same StartingCSV ??